elixir/apps/api/lib/api/controllers/fallback_controller.ex

```diff
@@ -1,5 +1,5 @@
 defmodule API.FallbackController do
-  use Phoenix.Controller
+  use API, :controller
```

Comment:
Seems like this was a typo, fixes a warning about the formats not being specified.